90.md

```diff
@@ -0,0 +1,438 @@
+        [ "bid", "100000" ]
+    ]
+}
+```
+
+### `kind:65002`: Job Request #2: summarization of job #1's result
+```json
+{
+    "id": "12346",
+    "pubkey": "abcdef",
+    "kind": 65003,
+    "content": "",
+    "tags": [
+        [ "i", "12345", "job" ], // input is the output of job with id 12345
+        [ "output", "text/plain" ],
+        [ "params", "length", "3 paragraphs" ],
+        [ "bid", "10000" ]
+    ]
+}
+```
+
+## Translation of a note
+### `kind:65004`: Job Request #1: translation of an existing note
+```json
+{
+    "id": "12346",
+    "pubkey": "abcdef",
+    "content": "",
+    "kind": 65004,
+    "tags": [
+        [ "i", "<hexid>", "event", "wss://relay.nostr.com" ]
+        [ "output", "text/plain" ],
+        [ "params", "lang", "es_AR" ],
+        [ "bid", "5000" ]
+    ]
+}
+```
+
+### `kind:65001`: Job result
+```json
+{
+    "kind": 65001,
+    "content": "Che, que copado, boludo!",
+    "tags": [
+        ["e", "12346"],
+        ["p", "abcdef"],
+        ["amount", "4000"]
+    ]
+}
+```
+
+## AI-image of the summarization of 2 podcasts
+
+### `kind:65002`: Job request #1 (transcribe podcast #1)
+```json
+{
+    "id": "123",
+    "kind": 65002,
+    "tags": [
+        [ "i", "https://example.com/episode1.mp3", "url" ],
+        [ "bid", "100000" ]
+    ]
+}
+```
+
+### `kind:65002`: Job request #2 (transcribe podcast #2)
+```json
+{
+    "id": "124",
+    "kind": 65002,
+    "tags": [
+        [ "i", "https://example.com/episode2.mp3", "url" ],
+        [ "bid", "100000" ]
+    ]
+}
+```
+
+### `kind:65003`: Job request #3 (summarize the two job's outputs into one paragraph)
+```json
+{
+    "id": "125",
+    "kind": 65003,
+    "tags": [
+        [ "param", "length", "1 paragraph" ],
+        [ "i", "123", "job" ],
+        [ "i", "124", "job" ],
+        [ "bid", "100000" ]
+    ]
+}
+```
+
+### `kind:65005`: Job request #4 (generate image based on the summary)
+```json
+{
+    "id": "126",
+    "kind": 65004,
+    "tags": [
+        [ "i", "125", "job" ],
+        [ "param", "prompt", "photorealistic" ],
+        [ "param", "size", "4000x4000" ],
+        [ "bid", "500000" ]
+    ]
+}
+```
+
+## AI-image of embedded input
+
+### `kind:65005`: Job request
+```json
+{
+    "kind": 65004,
+    "tags": [
+        [ "i", "Millions of vending machines, interconnected with tubes with eah other", "text" ],
+        [ "param", "prompt", "photorealistic" ],
+        [ "bid", "500000" ]
+    ]
+}
+```
+
+# Appendix 2: Job types
+
+This is a list of all the supported job requests.
+
+## speech-to-text: `kind:65002`
+
+### params
+
+| param      | req? | description                                               |
+|------------|------|-----------------------------------------------------------|
+| `range`    | opt  | timestamp range (in seconds) of desired text to be transcribed |
+| `alignment`| opt  | word, segment, raw:  word-level, segment-level, or raw outputs |
+
+## summarization: `kind:65003`
```

Comment:
"summary"?

Reply:
the AI people call it "summarization" 🤷‍♂️